docs/Flatpak_Build.md

```diff
@@ -0,0 +1,98 @@
+# PyBitmessage Linux flatpak instructions
+
+## Requirements
+First make sure you have `flatpak` and `flatpak-builder` installed. Follow the
+instructions for your distribution on [flathub](https://flatpak.org/setup/). The
+instructions there only cover the installation of `flatpak`, but 
+`flatpak-builder` should be the same.
+
```

Comment:
these are the packages (ubuntu 20.04) that need to be installed:

flatpak flatpak-builder git xz-utils elfutils librsvg2-common

Reply:
Okay, I'll update the documentation